src/components/Book.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  CircularProgressbar,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import '../styles/book.css';
+
+function Book() {
+  const percentage = 66;
+  return (
+    <div className="book-wrapper">
+      <div className="book-info">
+        <p className="book-genre">Action</p>
+        <p className="book-title">The Hunger games</p>
+        <p className="book-author">Sizanne collins</p>
+        <ul className="book-actions">
+          <li className="book-comment">Comments</li>
+          <li className="book-remove">Remove</li>
```

Comment:
- [ ] The remove needs to be a button, kindly update it

Reply:
Yeah sure done